github-and-markdown/github-and-markdown.md

```diff
@@ -0,0 +1,9 @@
+# Github-and markdown
+
+-Working easy with GitHub
+
+## Useful commands
+
+-git add
```

Comment:
that looks good but you also need the file-name?

Reply:
Thanks for the hint, i´ve changed the command.